Solutions/CohesitySecurity/SolutionMetadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "publisherId": "cohesity",
+    "offerId": "cohesity_sentinel_data_connector",
+    "firstPublishDate": "2022-10-10",
+    "providers": ["Cohesity"],
+    "categories": {
+        "domains": ["Security - Cloud Security", "Security - Automation (SOAR)"]
+    },
+    "support": {
+        "tier": "Partner",
+        "name": "Microsoft Corporation",
```

Comment:
@yinghuang123/ @kishannerella, can you please check and confirm on the support information? Should it be of cohesity?

Reply:
@v-sabiraj , that's what we originally thought, too but we were getting an error in one of the tools when we changed it to Cohesity.
That's why we followed the example that was suggested earlier in this thread (see https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/CiscoUmbrella/SolutionMetadata.json) and did the same as Cisco Umbrella, which eliminated the error.
```
{
	"publisherId": "azuresentinel",
	"offerId": "azure-sentinel-solution-ciscoumbrella",
	"firstPublishDate": "2022-04-01",
	"providers": [ "Cisco" ],
	"categories": {
		"domains": [ "Security - Cloud Security", "Security - Automation (SOAR)" ],
		"verticals": []
	},
	"support": {
		"tier": "Microsoft",
		"name": "Microsoft Corporation",
		"email": "support@microsoft.com",
		"link": "https://support.microsoft.com/"
	}
```

Reply:
Can you please check changing that, we will check on the error, thanks.

Reply:
@v-sabiraj , we retested, and now the package tool doesn't give an error. So, we put Cohesity info.